backend/app/management/commands/syncdb.py

```diff
@@ -44,9 +44,9 @@ def add_arguments(self, parser):
         parser.add_argument('--range', action='store', type=str)
 
     def handle(self, *args, **options):
-        spreadsheet_range = 'Photo!A:D'
+        spreadsheet_ranges = ['MapSquare', 'Photographer', 'Photo']
```

Comment:
It looks to me like in order for this to run correctly, this order is important. Please add a comment here specifying this, so later readers don't accidentally change the order values.